manticore/ethereum.py

```diff
@@ -918,7 +918,7 @@ def transaction(self, caller, address, value, data):
 
         return status
 
-    def multi_tx_analysis(self, solidity_filename, contract_name=None, tx_limit=None, tx_account="attacker"):
+    def multi_tx_analysis(self, solidity_filename, contract_name=None, tx_limit=None, tx_stop_using_coverage=True, tx_account="attacker"):
```

Comment:
This double negative is confusing. What if this was `use_coverage` and the check below was
`while current_coverage < 100 and use_coverage:`.

Is the default here to ignore coverage?

Reply:
Default is to use coverage to stop. I commit a few changes, let me know if it is easier to understand now..